data/json/vehicles/military.json

```diff
@@ -692,121 +313,50 @@
-      { "x": -2, "y": 1, "part": "hdframe_horizontal_2" },
-      { "x": -2, "y": 1, "part": "trunk" },
-      { "x": -2, "y": 1, "part": "roof" },
-      { "x": -2, "y": 2, "part": "hdframe_horizontal_2" },
-      { "x": -2, "y": 2, "part": "trunk" },
-      { "x": -2, "y": 2, "part": "hdroof" },
-      { "x": -2, "y": 3, "part": "hdframe_vertical" },
-      { "x": -2, "y": 3, "part": "hdboard_vertical" },
-      { "x": -2, "y": 3, "part": "plating_military" },
-      { "x": -2, "y": 0, "part": "hdframe_horizontal_2" },
-      { "x": -2, "y": 0, "part": "trunk" },
-      { "x": -2, "y": 0, "part": "hdroof" },
-      { "x": -2, "y": -1, "part": "hdframe_vertical" },
-      { "x": -2, "y": -1, "part": "hdboard_vertical" },
-      { "x": -2, "y": -1, "part": "plating_military" },
-      { "x": -3, "y": 1, "part": "hdframe_horizontal" },
-      { "x": -3, "y": 1, "part": "hddoor_trunk" },
-      { "x": -3, "y": 1, "part": "plating_military" },
-      { "x": -3, "y": 2, "part": "hdframe_horizontal" },
-      { "x": -3, "y": 2, "part": "hddoor_trunk" },
-      { "x": -3, "y": 2, "part": "plating_military" },
-      { "x": -3, "y": 3, "part": "hdframe_se" },
-      { "x": -3, "y": 3, "part": "hdboard_se" },
-      { "x": -3, "y": 3, "part": "plating_military" },
-      { "x": -3, "y": 3, "part": "wheel_armor" },
-      { "x": -3, "y": 0, "part": "hdframe_horizontal" },
-      { "x": -3, "y": 0, "part": "hddoor_trunk" },
-      { "x": -3, "y": 0, "part": "muffler" },
-      { "x": -3, "y": 0, "part": "plating_military" },
-      { "x": -3, "y": -1, "part": "hdframe_sw" },
-      { "x": -3, "y": -1, "part": "hdboard_sw" },
-      { "x": -3, "y": -1, "part": "plating_military" },
-      { "x": -3, "y": -1, "part": "wheel_armor" }
+      { "y": 3, "x": 3, "parts": [ "hdframe_horizontal", "plating_military" ] },
+      { "y": 2, "x": 3, "parts": [ "hdframe_horizontal_2", "plating_military" ] },
+      { "y": 1, "x": 3, "parts": [ "hdframe_horizontal_2", "plating_military" ] },
+      { "y": 0, "x": 3, "parts": [ "hdframe_horizontal_2", "plating_military" ] },
+      { "y": -1, "x": 3, "parts": [ "hdframe_horizontal", "plating_military" ] },
+      { "y": 3, "x": 2, "parts": [ "hdframe_ne", "hdhalfboard_ne", "plating_military" ] },
+      { "y": 3, "x": 2, "parts": [ "wheel_armor_steerable" ] },
+      { "y": 2, "x": 2, "parts": [ "hdframe_horizontal", "hdhalfboard_horizontal", "plating_military" ] },
+      { "y": 2, "x": 2, "parts": [ "headlight_reinforced" ] },
+      { "y": 1, "x": 2, "parts": [ "hdframe_horizontal", "hdhalfboard_horizontal", "plating_military" ] },
+      { "y": 1, "x": 2, "parts": [ "diesel_engine_v8", "alternator_truck", "battery_car" ] },
+      { "y": 0, "x": 2, "parts": [ "hdframe_horizontal", "hdhalfboard_horizontal", "plating_military" ] },
+      { "y": 0, "x": 2, "parts": [ "headlight_reinforced" ] },
+      { "y": -1, "x": 2, "parts": [ "hdframe_ne", "hdhalfboard_ne", "plating_military" ] },
```

Comment:
You gave the Humvee two "NE" frames

![image](https://user-images.githubusercontent.com/1426680/54856994-dcf36c00-4cdb-11e9-90e7-84baeb20ad54.png)